### PR TITLE
Here be the changelog

### PR DESCRIPTION
**Pull request by knightsamar**
Necessary so that cookbook-uploader script can process PRs on this cookbook
like on others.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+osl-mysql CHANGELOG
+===================
+This file is used to list changes made in each version of the
+osl-mysql cookbook.
+
+1.0.5
```
Comment (DesertBeagle) [inline CHANGELOG.md]: Shouldn't there be a date on this?
(2017-02-03)

Comment (knightsamar) [inline CHANGELOG.md]: @DesertBeagle Thanks for catching that! Fixed.

**Comment by knightsamar** [comment]
!bump patch ~

**Comment by knightsamar** [comment]
@ramereth this doesn't result in a chef version bump too. Do I need to keep CHANGELOG empty ?

**Comment by ramereth** [comment]
@knightsamar it might have been needed initially before being merged. At this point just manually bump it, etc.

**Pull request merged**